presto-main/src/main/java/com/facebook/presto/sql/planner/iterative/rule/CanonicalizeExpression.java

```diff
@@ -30,9 +30,9 @@
 
 import java.util.Optional;
 
-public class CanonicalizeExpressionRewriter
+public class CanonicalizeExpression
```

Comment:
Instead of renaming the class, I would rename the method from `rewrite` to `canoncalizeExpression`, then in case of static import it will readable what this method does.